--- buckets_sax.c	2007-04-17 20:54:49.000000000 +0200
+++ buckets_sax.c.oden	2009-01-07 11:08:44.000000000 +0100
@@ -77,7 +77,7 @@ static void sax_event_debug_log(const ch
     request_rec *r = bs->bctx->r_log;
 
     if (r)
-        ap_log_rerror(file, line, APLOG_DEBUG, 0, r,
+        ap_log_rerror(file, line, APLOG_DEBUG, 0, r, "%s", 
                       apr_pvsprintf(bs->bctx->p_tmp, msg, args));
 
     va_end(args);
